configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
On Fedora, the gtest-devel package installs /usr/share/aclocal/gtest.m4... the GTEST_LIB_CHECK macro is defined in that file.  If gtest.m4 is not installed from a source install or other distros don't include it in a package, then we may need to provide it in libyami's m4 directory.  Are you able to check if other distros provide this?